src/patches/dense_patch_computation.cpp

```diff
@@ -218,10 +218,21 @@ InstructionApplicationResult try_apply_local_instruction(
         slice.patch_at(move->source_cell)->id = std::nullopt;
         return {nullptr, {}};
     }
-    else if (std::holds_alternative<LocalInstruction::TwoPatchMeasure>(instruction.operation))
+    else if (const auto* localmeas = std::get_if<LocalInstruction::TwoPatchMeasure>(&instruction.operation))
     {
-        LSTK_NOT_IMPLEMENTED;
-    }
+        if (slice.patch_at(localmeas->cell1)->is_active())
+        {
+            return {std::make_unique<std::runtime_error>(lstk::cat("Patch at ", localmeas->cell1, " is active")), {}};
```

Comment:
I started the convention of prefixing the errors with the instruction, to make it easier to debug
```suggestion
            return {std::make_unique<std::runtime_error>(lstk::cat(instruction, ": Patch at ", localmeas->cell1, " is active")), {}};
```